### DISCUSSION TITLE
Can I include xtraplatform dependencies, such as xtraplatform-core, in LDProxy and run it locally?

### DISCUSSION THREAD

**Discussion by dallaslynnlin** [ldproxy/ldproxy#999] [Setup] [answered]
I'm currently working with LDProxy and I would like to include specific dependencies from the xtraplatform module, such as xtraplatform-core, in my local setup. This would allow me to run LDProxy with the modified code. I need guidance on how to modify the relevant file(s) to achieve this, as well as how to obtain the xtraplatform-core files,Is this the path: "../../../../../xtraplatform/xtraplatform-core/xtraplatform-manager/src/main/javascript/core"?

**Comment by azahnen** [comment, marked as answer]
You can just clone the `xtraplatform` repository as a sibling directory to `ldproxy`. When you then use the gradle project in `ldproxy/composite` instead of `ldproxy`, it will automatically include the local `xtraplatform` code instead of the published jars.
